pkg/ui/react-app/src/pages/graph/PanelList.tsx

```diff
@@ -114,6 +118,7 @@ const PanelList: FC<RouteComponentProps & PathPrefixProps> = ({ pathPrefix = ''
   const [enableQueryHistory, setEnableQueryHistory] = useLocalStorage('enable-query-history', false);
 
   const { response: metricsRes, error: metricsErr } = useFetch<string[]>(`${pathPrefix}/api/v1/label/__name__/values`);
+  const { response: storesRes, error: storesErr, isLoading: storesLoading } = useFetch<StoreListProps>(`${pathPrefix}/api/v1/stores`);
```

Comment:
We need to show an alert in case of an error here as we are doing for `metricsErr`.